lib/WebpackOptionsDefaulter.js

```diff
@@ -81,7 +81,7 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 
 		this.set("resolve", {});
 		this.set("resolve.unsafeCache", true);
-		this.set("resolve.modules", ["node_modules"]);
+		this.set("resolve.modules", ["node_modules", process.env.NODE_PATH].filter(Boolean));
```

Comment:
hehe, true, it could have been easy for once :P.

I guess we need to check for `:` (or `;` on windows machines?)

will add that tonight